webapp/src/Serializer/SubmissionVisitor.php

```diff
@@ -85,13 +85,13 @@ public function onPostSerialize(ObjectEvent $event)
             /** @var Submission $submission */
             $submission         = $event->getObject();
             $filesRoute         = $this->router->generate(
-                'submission_files',
+                'v4_submission_files',
```

Comment:
This name (and the below one) changed because of the name_prefix in the YAML above